README.md

```diff
@@ -9,6 +9,13 @@ A convenient command-line tool helping you keep repositories clean.
 # pip install git-delete-merged-branches
 ```
 
+If you are using one of the GNU/Linux distributions below, you can install
+`git-delete-merged-branches` through the respective package manager, e.g.:
+
+
+ - [Arch Linux (AUR](https://aur.archlinux.org/packages/git-delete-merged-branches/):
```

Comment:
There is a round closing brace missing after `(AUR`. Could you add it?

While at it, could you replace the `: ` by ` — ` (emdash)? It would look less crowded I believe. No?

Reply:
> There is a round closing brace missing after (AUR. Could you add it?

Ah silly mistake :see_no_evil: sure.

> While at it, could you replace the : by — (emdash)? It would look less crowded I believe. No?

I'm not entirely sure what you mean with less crowded, but see my edit :slightly_smiling_face: